daemon/inspect.go

```diff
@@ -151,3 +166,30 @@ func (daemon *Daemon) VolumeInspect(name string) (*types.Volume, error) {
 	}
 	return volumeToAPIType(v), nil
 }
+
+func getBackwardsCompatibleNetworkSettings(settings *network.Settings) *v1p20.NetworkSettings {
```

Comment:
Yes, but if some use did  the following:
```
docker run --net=somenetwork -d busybox
docker inspect --format '{{.NetworkSettings.IPAddress}}' <container_id>
```
it will return an empty value. Is that acceptable?
Or are you trying to deprecate the old fields this way? 

P.S: this will affect not just a overlay network but also a regular bridge network which is non-default